drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -808,6 +836,18 @@ void RasterizerCanvasGLES3::_render_items(RID p_to_render_target, int p_item_cou
 	state.current_batch_index = 0;
 	state.canvas_instance_batches.clear();
 	state.last_item_index += index;
+
+	// Blit MSAA 2D Backbuffer to normal backbuffer
+	// to support sampling from backbuffer
+	if (p_to_backbuffer && render_target->msaa_2d.mode != RS::VIEWPORT_MSAA_DISABLED && render_target->msaa_2d.backbuffer_fbo != 0 && render_target->msaa_2d.needs_resolve) {
+		glBindFramebuffer(GL_READ_FRAMEBUFFER, render_target->msaa_2d.backbuffer_fbo);
+		glBindFramebuffer(GL_DRAW_FRAMEBUFFER, render_target->backbuffer_fbo);
+
+		glBlitFramebuffer(0, 0, render_target->size.x, render_target->size.y,
+				0, 0, render_target->size.x, render_target->size.y,
+				GL_COLOR_BUFFER_BIT, GL_NEAREST);
+		glBindFramebuffer(GL_FRAMEBUFFER, render_target->msaa_2d.backbuffer_fbo);
+	}
```

Comment:
This can happen several times for each call to `canvas_render_items()`. Which itself can be called several times per frame. This shouldn't be happening so frequently.